### PR TITLE
updated PR template

### PR DESCRIPTION
**Pull request by SvA1**

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
+# Please make sure you have done the following:
 
-- [ ] Created a folder with the name of language you code inside `/Day<n>/` directory (Where n is for nth day)
-- [ ] Added your Codes
+- [ ] Created a folder with the name of language you code inside directory for the day you are making the contribution to
+- [ ] Added your codes
 - [ ] Tested your codes
 - [ ] Added credits (author and date) at the top of the code
-- [ ] Updated README.md
+- [ ]  Updated README.md of the day you are contributing to
```
Comment (MadhavBahlMD) [inline PULL_REQUEST_TEMPLATE.md:7]:
```suggestion
- [ ] Updated README.md of the day you are contributing to
```

Why do we need an extra space here?

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-Please make sure you have done the following:
+# Please make sure you have done the following:
 
-- [ ] Created a folder with the name of language you code inside `/Day<n>/` directory (Where n is for nth day)
-- [ ] Added your Codes
+- [ ] Created a folder with the name of language you code inside directory for the day you are making the contribution to
```
Comment (aaditkamat) [inline PULL_REQUEST_TEMPLATE.md:3]: Seems like the earlier version was clearer.

File: PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
+- [ ] Added your codes
 - [ ] Tested your codes
 - [ ] Added credits (author and date) at the top of the code
-- [ ] Updated README.md
+- [ ] Updated README.md of the day you are contributing to
```
Comment (aaditkamat) [inline PULL_REQUEST_TEMPLATE.md:7]:
```suggestion
- [ ] Updated README.md for `/Day <n>/`
```

File: PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
-Please make sure you have done the following:
+# Please make sure you have done the following:
 
-- [ ] Created a folder with the name of language you code inside `/Day<n>/` directory (Where n is for nth day)
-- [ ] Added your Codes
+- [ ] Created a folder with the name of language you code inside directory for the day you are making the contribution to
+- [ ] Added your codes
 - [ ] Tested your codes
```
Comment (aaditkamat) [inline PULL_REQUEST_TEMPLATE.md:5]:
```suggestion
- [ ] Tested your code
```

File: PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
-Please make sure you have done the following:
+# Please make sure you have done the following:
 
-- [ ] Created a folder with the name of language you code inside `/Day<n>/` directory (Where n is for nth day)
-- [ ] Added your Codes
+- [ ] Created a folder with the name of language you code inside directory for the day you are making the contribution to
+- [ ] Added your codes
```
Comment (aaditkamat) [inline PULL_REQUEST_TEMPLATE.md:4]:
```suggestion
- [ ] Added the source code file inside `/Day<n>/<Language>/`
```

**Review by aaditkamat — CHANGES_REQUESTED** [review]
These are my suggestions. @MadhavBahlMD, if you have any feedback?

**Comment by aaditkamat** [comment]
Fixes #347

File: PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (MadhavBahlMD) [reply PULL_REQUEST_TEMPLATE.md:3]: This is perfect

Thanks a lot @aaditkamat 

@SvA1 please make these changes

File: PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (MadhavBahlMD) [reply PULL_REQUEST_TEMPLATE.md:7]: Looks good

**Comment by MadhavBahlMD** [comment]
@SvA1 please make the changes requested by @aaditkamat

**Pull request merged**